Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -5,16 +5,35 @@ import Turf
 
 // swiftlint:disable file_length
 
-public enum PreferredFPS: Int, Equatable {
+public enum PreferredFPS: Equatable {
```

Comment:
Would be best to just make `PreferredFPS` conform to `RawRepresentable` -- instead of adding a separate `fps` var.